docs/user_guide/request_cancellation.md

```diff
@@ -28,7 +28,7 @@
 
 # Request Cancellation
 
-Starting from r23.10, Triton supports handling request cancellation received
+Starting from r23.11, Triton supports handling request cancellation received
```

Comment:
I think this should stay 23.10 CC @tanmayv25

Reply:
Addressed